br/tests/lightning_tikv_multi_rocksdb/run.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+cur=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+. $cur/../_utils/run_services
+export TIKV_CONFIG="tests/config/tikv-multi-rocksdb.toml"
+start_services
```

Comment:
Is `start_services` called twice? One is when we execute `br/tests/run.sh`, and the other is here.

Reply:
https://github.com/pingcap/tidb/pull/42243/commits/ef2fae904859247088db27ca36655acd70cd72b0